content/200-orm/500-reference/200-prisma-cli-reference.mdx

```diff
@@ -135,6 +135,8 @@ The `init` command does not interpret any existing files. Instead, it creates a
 | `--generator-provider`  | No       | Define the generator provider to use.                                                                                                                                                                                               | `prisma-client-js`            |
 | `--preview-feature`     | No       | Define the [Preview features](/orm/reference/preview-features) to use. To define multiple Preview features, you have to provide the flag multiple times for each Preview feature. [See example](#run-prisma-init---preview-feature) |                               |
 | `--output`              | No       | Specifies the [output location for the generated client](/orm/prisma-client/setup-and-configuration/generating-prisma-client#using-a-custom-output-path).                                                                           | `node_modules/.prisma/client` |
+| `--with-model`              | No       | Adds a simple `User` model to the initial Prisma schema |
```

Comment:
from @Jolg42 's suggestion
```suggestion
| `--with-model`              | No       | Adds a simple `User` model to the initial Prisma schema. Available since version `5.14.0`. |
```